src/DataWrangling/ECCO/ECCO.jl

```diff
@@ -90,7 +93,7 @@ const ECCO_z = [
 
 empty_ECCO_field(variable_name::Symbol; kw...) = empty_ECCO_field(ECCOMetadata(variable_name); kw...)
 
-function empty_ECCO_field(metadata::ECCOMetadata;
+function empty_ECCO_field(metadata::Union{ECCOMetadata,ECCODarwinMetadata};
```

Comment:
If you switch to implementing a version instead of a metadata object this change (and similar ones below) are not required